openshift/template.yaml

```diff
@@ -196,83 +41,3 @@ objects:
           name: cad-pd-token
       image: ${REGISTRY_IMG}:${IMAGE_TAG}
```

Comment:
we need to keep this a template i think 
app interface builds our image and provides the image tag for us as env
https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/docs/app-sre/continuous-delivery-in-app-interface.md#automatically-generated-parameters